sdks/java/extensions/join-library/README.md

```diff
@@ -22,12 +41,4 @@ Example how to use join-library:
     PCollection<KV<String, KV<String, Long>>> joinedPcollection =
       Join.innerJoin(leftPcollection, rightPcollection);
 
-Join-library can be found on maven-central:
-
-    <dependency>
-      <groupId>org.linuxalert.dataflow</groupId>
-      <artifactId>google-cloud-dataflow-java-contrib-joinlibrary</artifactId>
-      <version>0.0.3</version>
-    </dependency>
-
```

Comment:
It will, but not with these Maven coordinates: it will use the Beam ones. Let me update.